task.sql

```diff
@@ -2,0 +1,38 @@
+CREATE DATABASE ShopDB;
+
+USE ShopDB;
+
+CREATE TABLE Products (
+    ProductID INT AUTO_INCREMENT,
+    Name VARCHAR(50),
+    Description VARCHAR(50),
+    Price VARCHAR(50),
+    WarehouseAmount VARCHAR(50),
+    PRIMARY KEY (ProductID)
```

Comment:
The 'WarehouseAmount' column is defined as VARCHAR(50). This should likely be an integer type, such as INT, to accurately represent the quantity of products available in the warehouse.